homeassistant/components/islamic_prayer_times/coordinator.py

```diff
@@ -88,60 +88,53 @@ def get_new_prayer_times(self) -> dict[str, Any]:
     def async_schedule_future_update(self, midnight_dt: datetime) -> None:
         """Schedule future update for sensors.
 
-        Midnight is a calculated time.  The specifics of the calculation
-        depends on the method of the prayer time calculation.  This calculated
-        midnight is the time at which the time to pray the Isha prayers have
-        expired.
+        The least surprising behaviour is to load the next day's prayer times only
+        after the current day's prayers are complete. We will take the fiqhi opinion
+        that Isha should be prayed before Islamic midnight (which may be before or after 12:00 midnight),
```

Comment:
Please limit docstring lines to max 72 characters per line in accordance with PEP8.